paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -198,16 +199,10 @@ internal class PaymentSheetActivityTest {
 
     @Test
     fun `updates buy button state on add payment`() {
-        val scenario = activityScenario()
+        val viewModel = createViewModel(paymentMethods = emptyList())
+        val scenario = activityScenario(viewModel)
         scenario.launch(intent).onActivity { activity ->
-            // Based on previously run tests the viewModel might have a different selection state saved
-            viewModel.updateSelection(null)
-
-            viewModel.transitionTo(
-                PaymentSheetViewModel.TransitionTarget.AddPaymentMethodFull(
-                    FragmentConfigFixtures.DEFAULT
-                )
-            )
```

Comment:
This is no longer necessary now that we use a new view model instance for each test, as this code is already executed when launching the activity.